File: DotNet/CesiumLanguageWriter/Advanced/ICesiumDoubleListValuePropertyWriter.cs
using System.Collections.Generic;

namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a list of double values.
    /// </summary>
    public interface ICesiumDoubleListValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value as a list of double values.
        /// </summary>
        /// <param name="values">The values.</param>
        void WriteArray(IEnumerable<double> values);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumDoubleListValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;

/**
 * A writer that can write a value as a list of double values.
 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public interface ICesiumDoubleListValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value as a list of double values.
    * @param values The values.
    */
    void writeArray(Iterable<Double> values);
}